console/src/main/webapp/manager/app/components/user/user.tpl.html

```diff
@@ -202,7 +202,7 @@ <h5 ng-if="user.message.attachments.length>0" translate>msg.attachments</h5>
             <tr ng-repeat="message in user.messages.emails">
               <td>{{::message.date | date: 'mediumDate'}}</td>
               <td>
-                <a ng-click="user.openMessage(message)" href="javascript:void(0);">
+                <a ng-link="user({id: user.uid, tab: 'messages', queryParams: {msgid: message.id}})">
```

Comment:
I can't open message. Link seems to be broken because of no action on click.
It works if i replace new : 
`<a ng-link="user({id: user.uid, tab: 'messages', queryParams: {msgid: message.id}})">`
By old :
`<a ng-click="user.openMessage(message)" href="javascript:void(0);">`

Reply:
But all seems to be good from logger component. Message could be open by click on the message log.